bootstrap/shell/shell.go

```diff
@@ -169,6 +169,18 @@ func (s *Shell) Terminate() {
 	}
 }
 
+// Terminate running command
+func (s *Shell) WaitStatus() process.WaitStatus {
+	s.cmdLock.Lock()
+	defer s.cmdLock.Unlock()
+
+	if s.cmd != nil && s.cmd.proc != nil {
+		return s.cmd.proc.WaitStatus()
+	}
+	var ws syscall.WaitStatus
```

Comment:
The type difference between platforms feels worthy of a small explanatory comment.